src/enums/entity-types.ts

```diff
@@ -23,7 +23,6 @@ export enum EntityTypes {
   SP = CorpTypeCd.SOLE_PROP,
   UL = CorpTypeCd.UNLIMITED_LIABILITY_COMPANY,
   ULC = CorpTypeCd.BC_ULC_COMPANY,
-  C = CorpTypeCd.CONTINUE_IN,
```

Comment:
I'm surprised this duplicate key didn't cause an error, or did it? Anyway, good fix 👍